### ISSUE TITLE
IR Air Condition not support!

### ISSUE THREAD

**Issue by Serjaru** [openshwprojects/OpenBK7231T_App#841]
IR Air Condition not support! Only TV! Look as in Tasmota!

**Comment by openshwprojects** [comment]
You are correct. You are most likely asking for the feature included in this PR:
https://github.com/openshwprojects/OpenBK7231T_App/pull/723
The PR is almost ready but is awaiting for some final stability fixes. Still, you can get binaries from it and try yourself if it works for you.
